mongodb_exporter.go

```diff
@@ -86,8 +90,16 @@ func registerCollector() {
 
 func main() {
 	flag.Parse()
+
+	if *printVersion {
+		fmt.Printf("mongodb_exporter version: %s\n", version)
+		fmt.Printf("git commit hash: %s\n", versionGitCommit)
+		os.Exit(0)
+	}
+
 	shared.ParseEnabledGroups(*enabledGroupsFlag)
 
 	fmt.Println("### Warning: the exporter is in beta/experimental state and field names are very\n### likely to change in the future and features may change or get removed!\n### See: https://github.com/Percona-Lab/prometheus_mongodb_exporter for updates")
+	fmt.Printf("Starting mongodb_exporter version: %s, git commit: %s\n", version, versionGitCommit)
```

Comment:
Why not just make this and 94-98 share a common function :) And make them both report on a single line.